integreat_cms/api_client_firebase/push_notification_sender.py

```diff
@@ -0,0 +1,156 @@
+
+import logging
+import requests
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from ..cms.forms.push_notifications.push_notification_translation_form import PushNotificationTranslation
```

Comment:
```suggestion
from ..cms.forms.push_notifications.push_notification_translation_form import (
    PushNotificationTranslation,
)
```
Our automatic formatting tool wants to change these lines.
Please run `./dev-tools/black.sh` before you push your code or execute `pipenv run pre-commit install` do enable the pre commit hooks which do this automatically before every commit.